Silicon/TigerlakePkg/Include/CpuPowerMgmt.h

```diff
@@ -0,0 +1,237 @@
+/** @file
+  This file contains define definitions specific to processor
+
+  Copyright (c) 1999 - 2019, Intel Corporation. All rights reserved.<BR>
+  SPDX-License-Identifier: BSD-2-Clause-Patent
+**/
+#ifndef _POWER_MGMT_DEFINITIONS_H_
+#define _POWER_MGMT_DEFINITIONS_H_
+
+
+//
+// Voltage offset definitions
+//
+#define OC_LIB_OFFSET_ADAPTIVE  0
+#define OC_LIB_OFFSET_OVERRIDE  1
+//
+// Platform Power Management Flags Bit Definitions:
+//   These defines are also used in CPU0CST.ASL to check platform configuration
+//   and build C-state table accordingly.
+//
+#ifdef __GNUC__
+//
+// @todo Add workaround for GCC build, In GCC build the BIT can not be changed to value by GCC (ex: BIT0 can not be changed to 0x1)
+//
+#define PPM_EIST                0x1        ///< BIT 0 : Enhanced Intel Speed Step Technology.
+#define PPM_C1                  0x2        ///< BIT 1 : C1 enabled, supported.
+#define PPM_C1E                 0x4        ///< BIT 2 : C1E enabled.
+#ifdef CPU_CFL

```

Comment:
Do we need to keep CPU_CFL?

Reply:
Noted, I will remove all the #ifndef & #endif for CPU_CFL